### PR TITLE
Added ‘Show Toolbar’ permission.

### PR DESCRIPTION
**Pull request by agitator**
merge with https://github.com/plone/Products.CMFPlone/pull/2100 and https://github.com/plone/plone.app.upgrade/pull/127

### CODE REVIEW

File: plone/app/layout/permissions.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('plone.app.layout')
+
+security.declarePublic('ShowToolbar')
```
Comment (gforcada) [inline plone/app/layout/permissions.py]: Just a wild guess, but if this is the 'id' of the permission, shouldn't it be 'plone.showToolbar' like on the ZCML?

Comment (datakurre) [inline plone/app/layout/permissions.py]: @gforcada I believe that just defines the module level attribute ``ShowToolbar`` defined below to be public.

But why is this ``permissions.py`` required at all? @agitator? I don't see it being imprted anywhere. And default roles for that permission are set in separate pulls for CMFPlone and plone.app.upgrade.

Comment (agitator) [inline plone/app/layout/permissions.py]: maybe is just confused with the various types on permission definitions in CMFPlone, CMFCore, ...

File: CHANGES.rst [hunk 2]
```diff
@@ -10,7 +10,8 @@ Breaking changes:
 
 New features:
 
-- *add item here*
+- Added ‘Show Toolbar’ permission.
```
Comment (mauritsvanrees) [inline CHANGES.rst]: I fear that these pretty quotes could trip up zest.releaser. Could you make them normal single quotes? Or maybe `` to display it as code, or `*` to display it cursive.

Comment (agitator) [inline CHANGES.rst]: done, changed it to ``

**Comment by thet** [comment]
LGTM

**Pull request merged**